core/src/main/java/io/confluent/kafka/schemaregistry/rest/client/LocalSchemaRegistryClient.java

```diff
@@ -274,11 +273,9 @@ public synchronized List<ParsedSchema> getSchemas(
     } catch (SchemaRegistryException e) {
       throw Errors.schemaRegistryException(errorMessage, e);
     }
-    while (schemas.hasNext()) {
-      Schema s = schemas.next();
-      result.add(parseSchema(s).get());
-    }
-    return result;
+    return schemas.stream()
```

Comment:
You can convert an iterator to a stream using `Streams.stream(iterator)`

See
https://www.javadoc.io/doc/com.google.guava/guava/latest/com/google/common/collect/Streams.html#stream(java.util.Iterator)